package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mybrand-api",
+  "version": "1.0.0",
+  "description": "this is a backend version of my brand",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha test/project-setup-test.js",
+    "dev-start": "babel-watch server.js",
```

Comment:
Remove this one and instead add a start script (that doesn't use nodemon; this will be used in production) that will run your entry file in the build folder.